### ISSUE TITLE
'ERR_UNKNOWN_FILE_EXTENSION'

### ISSUE THREAD

**Issue by aniyazngn** [jeffijoe/typesync#136]
Getting this error 'ERR_UNKNOWN_FILE_EXTENSION' on node  v20.8.0 

Does anyone have a fix or this package will get an update on?

**Comment by jeffijoe** [comment]
@lishaduck do you think this is related to the ESM change from #134 ?

**Comment by lishaduck** [comment]
Are you running latest typesync 0.14.0?
Does updating Node.js help?
Are you in a monorepo?
Do you have `"type": "module"` set in your `package.json` file?
Are you invoking it via `npx` or is it globally installed?
What platform are you on? Windows?

**Comment by aniyazngn** [comment]
> @aniyazngn,
> 
> Are you running latest typesync 0.14.0? Does updating Node.js help? Are you in a monorepo? Do you have `"type": "module"` set in your `package.json` file? Are you invoking it via `npx` or is it globally installed? What platform are you on? Windows?

Yes tried updating Node to latest did not help!
yes currently on windows ,its the same on mac as well
i was invoking through npx and then tried manually but it was not helpful , keep throwing the same error but as soon as i change node version to 18 it works perfectly!
sorry for late reply was busy working on this project in the end i removed typesync and there was no errors
i am assuming this has to do with node version 20.0 and above cus of the typesync package does not have a proper file extension

**Comment by lishaduck** [comment]
> Yes tried updating Node to latest did not help!
yes currently on windows ,its the same on mac as well i was invoking through npx and then tried manually but it was not helpful , keep throwing the same error but as soon as i change node version to 18 it works perfectly! sorry for late reply was busy working on this project in the end i removed typesync and there was no errors i am assuming this has to do with node version 20.0 and above cus of the typesync package does not have a proper file extension

Hmm. I'm on Node 22.10 on a mac, installed globally. Works on my machine 😁
Passes CI (Linux, Node 22.12.0) https://github.com/PSDTools/psdtools.github.io/pull/127 & https://github.com/AndreaPontrandolfo/sheriff/pull/312.

**Comment by lishaduck** [comment]
@aniyazngn, can you copy+paste the _full_ error message? I can't help if I don't know what file is unknown.

**Comment by lishaduck** [comment]
@aniyazngn, ping :)
Can you paste the full stacktrace? I still can't repro.

**Comment by jeffijoe** [comment]
Closing due to inactivity
